examples/qbv-example/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
+	repositories {
+		mavenLocal()
+		mavenCentral()
+	}
+	dependencies {
+		classpath "com.marklogic:ml-gradle:4.6-SNAPSHOT"
+	}
+}
+
+plugins {
+	id "net.saliman.properties" version "1.5.1"
+	id "java"
+}
+
+apply plugin: "com.marklogic.ml-gradle"
+
+repositories {
+	mavenLocal()
+	mavenCentral()
+	maven { url "https://developer.marklogic.com/maven2/" }
+}
+
+configurations {
+	mlcp
+}
+
+dependencies {
+	mlcp 'com.marklogic:mlcp:10.0.9.5'
+	mlcp files("lib")
+}
+
+/**
+ * Using an MlcpTask to load the data into the content database.
+ * See the mlcp example project (examples/mlcp-project) for more information on this technique.
+ */
+task importEmployeeData(type: com.marklogic.gradle.task.MlcpTask) {
+	classpath = configurations.mlcp
+	command = "IMPORT"
+	database = mlAppConfig.contentDatabaseName
+	output_collections = "sample-import"
+	output_permissions = "rest-reader,read,rest-writer,update"
+	output_uri_replace = ".*import,'/import'"
+	input_file_type = "delimited_text"
+	delimited_root_name = "Employee"
+	args = ["-input_file_path", "data"]
+}
+
+/**
+ * Using a ServerEvalTask to evaluate a JS script on the server in order to test the Engineering QBV.
+ */
+task getEngineeringEmployees(type: com.marklogic.gradle.task.ServerEvalTask) {
+	description = "Run this to verify that the plugin from the example-dependency bundle was installed correctly; should get 66.67 as a result"
```

Comment:
Copy/pasta! Replace this with the text in the comment and that remove the comment, same for the task below.